mithril-client/src/services/mithril_stake_distribution.rs

```diff
@@ -94,17 +94,17 @@ impl AppMithrilStakeDistributionService {
     // TODO: Create a structure to handle message verification.
     async fn create_clerk(
         &self,
-        signers_with_stake: &[SignerWithStake],
-        protocol_parameters: &ProtocolParameters,
+        stake_distribution: &MithrilStakeDistribution,
     ) -> StdResult<Option<ProtocolClerk>> {
```

Comment:
The signature of this method should probably not be changed so that we can factor it easily later?

Reply:
I meant that if the signature of the function changes, it will not help the work that will be done in #669